sysdig/resource_sysdig_user_test.go

```diff
@@ -48,35 +50,35 @@ func TestAccUser(t *testing.T) {
 func userWithName(name string) string {
 	return fmt.Sprintf(`
 resource "sysdig_user" "sample" {
-  email      = "terraform-test+user@sysdig.com"
+  email      = "terraform-test+user%[1]s@sysdig.com"
   system_role = "ROLE_USER"
-  first_name = "%s"
-  last_name  = "%s"
-}`, name, name)
+  first_name = "%[1]s"
+  last_name  = "%[1]s"
+}`, name)
 }
 
 func userWithSystemRole(name string) string {
 	return fmt.Sprintf(`
 resource "sysdig_user" "sample" {
-  email      = "terraform-test+user@sysdig.com"
-  system_role = "ROLE_CUSTOMER"
-  first_name = "%s"
-  last_name  = "%s"
-}`, name, name)
+  email      = "terraform-test+user%[1]s@sysdig.com"
+  system_role = "ROLE_USER"
+  first_name = "%[1]s"
+  last_name  = "%[1]s"
+}`, name)
 }
 
 func userWithoutSystemRole(name string) string {
 	return fmt.Sprintf(`
 resource "sysdig_user" "sample" {
-  email      = "terraform-test+user@sysdig.com"
-  first_name = "%s"
-  last_name  = "%s"
+  email      = "terraform-test+user%[1]s@sysdig.com"
+  first_name = "%[1]s"
+  last_name  = "%[1]s"
 }`, name, name)
```

Comment:
The 3rd argument of `Sprintf` (i.e. the 2nd `name`) looks not necessary since we are using `%[1]s` always

Reply:
good catch